app/controllers/vendor_api/error_handling.rb

```diff
@@ -0,0 +1,32 @@
+require 'active_support/concern'
+
+module VendorApi::ErrorHandling
+  extend ActiveSupport::Concern
+
+  class InvalidMetadata < StandardError; end
+
+  included do
+    rescue_from ActiveRecord::RecordNotFound, with: :application_not_found
+    rescue_from InvalidMetadata, with: :invalid_metadata
```

Comment:
It looks like this line, the exception class and `invalid_metadata(_e)` aren't actually used?